README.md

```diff
@@ -1,4 +1,4 @@
-<img align="right" src="https://cdn.miranj.in/mc3/img/apple-touch-icon-precomposed.png" width="57" height="57" alt="Miranj">
+<img align="right" src="https://cdn.craft.cloud/032c8560-18a8-45ba-a958-a03804e5cb66/assets/partners/1146942.png" width="57" height="57" alt="Miranj">
```

Comment:
We should not depend on a file hosted on a third party server. That is the right image but the URL for this should be one that we have control over.

Reply:
Is there a place with a list of all branding resources with URLs from our CDN?